bindings/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pravega"
-version = "0.0.1-alpha-3"
+version = "0.0.1-alpha-4"
```

Comment:
The change will be published only in case we tag a branch ref: https://github.com/pravega/pravega-client-rust/blob/7a0cd9d217bd742824ff209a5c4d9c09847c12c2/.github/workflows/tagPublish.yml#L6 
This can be changed...

Reply:
Thanks. I increment it since some change has been made, so if we build it from scratch it would be different than the published one. Right now I'm not trying to publish it, only to build locally and test it.